.github/ISSUE_TEMPLATE/offboarding.md

```diff
@@ -41,20 +41,36 @@ ticket has been completed. At that time, put the ticket back to triage.
+- [ ] Created a new `anvilprod` GitLab EC2 instance using a private key from the replacement system administrator <sub>or the offboarded employee/contractor was not the sole system administrator</sub>
+- [ ] Added the replacement system administrator's Google workspace account to the `platform-hca-dev` Google project <sub>or the offboarded employee/contractor was not the sole system administrator</sub>
+- [ ] Added the replacement system administrator's Google workspace account to the `platform-hca-prod` Google project <sub>or the offboarded employee/contractor was not the sole system administrator</sub>
+- [ ] Added the replacement system administrator's Google workspace account to the `platform-anvil-dev` Google project <sub>or the offboarded employee/contractor was not the sole system administrator</sub>
+- [ ] Added the replacement system administrator's Google workspace account to the `platform-anvil-prod` Google project <sub>or the offboarded employee/contractor was not the sole system administrator</sub>
+- [ ] Added `Project Owner` permissions to the replacement system administrator's Google workspace account <sub>or the offboarded employee/contractor was not the sole system administrator</sub>
+- [ ] Added the replacement system administrator's Google workspace account to the `azul-dev` group in Terra <sub>or the offboarded employee/contractor was not the sole system administrator</sub>
+- [ ] Added the replacement system administrator's Google workspace account to the `azul-prod` group in Terra <sub>or the offboarded employee/contractor was not the sole system administrator</sub>
+- [ ] Added the replacement system administrator's Google workspace account to the `azul-anvil-dev` group in Terra <sub>or the offboarded employee/contractor was not the sole system administrator</sub>
+- [ ] Added the replacement system administrator's Google workspace account to the `azul-anvil-prod` group in Terra <sub>or the offboarded employee/contractor was not the sole system administrator</sub>
+- [ ] Set the `Can manage users (admin)` checkbox for the replacement system administrator in the `azul-dev` group in Terra <sub>or the offboarded employee/contractor was not the sole system administrator</sub>
+- [ ] Set the `Can manage users (admin)` checkbox for the replacement system administrator in the `azul-prod` group in Terra <sub>or the offboarded employee/contractor was not the sole system administrator</sub>
+- [ ] Set the `Can manage users (admin)` checkbox for the replacement system administrator in the `azul-anvil-dev` group in Terra <sub>or the offboarded employee/contractor was not the sole system administrator</sub>
+- [ ] Set the `Can manage users (admin)` checkbox for the replacement system administrator in the `azul-anvil-prod` group in Terra <sub>or the offboarded employee/contractor was not the sole system administrator</sub>
```

Comment:
An earlier version of this checklist had optional sections based on the role the offboarded person had, however this complicated the checklist and made it hard to follow, especially with the limited formatting & indentation options available via markdown in a GitHub issue.

Instead, like our PR checklists, it was decided to go with a linear list that could be completed top to bottom with every checkbox being processed.

To address the point you raised that "contractors never perform organization/system administrator role", I've re-worded the `<sub>` notes so that they don't imply a contractor was performing the system admin role.